src/modules/notifications/NotificationConstants.ts

```diff
@@ -57,7 +57,8 @@ const NotificationConstants = {
         },
         Items: {
             route_item_found: new NotificationSetting('notification.route_item_found', 'Item found during route battle', true),
-            dungeon_item_found: new NotificationSetting('notification.dungeon_item_found', 'Item found in dungeon chest', true),
+            common_dungeon_item_found: new NotificationSetting('notification.common_dungeon_item_found', 'Item found in dungeon chest', true),
+            rare_dungeon_item_found: new NotificationSetting('notification.rare_dungeon_item_found', 'Item found in dungeon chest', true),
```

Comment:
They currently have the same title.
Also, maybe we should have an update function for them, so people get to keep their setup?

Reply:
Changed title, unsure how to write the update function